drivers/include/bme680.h

```diff
@@ -211,111 +57,145 @@ typedef union {
 } bme680_intf_t;
 
 /**
- * @brief   BME680 device initialization parameters
+ * @brief   Device initialization parameters
  */
 typedef struct {
-    uint8_t ifsel;              /**< Interface selection */
-    uint8_t temp_os;            /**< Temperature oversampling */
-    uint8_t hum_os;             /**< Humidity oversampling */
-    uint8_t pres_os;            /**< Pressure oversampling */
-    uint8_t filter;             /**< IIR filter coefficient */
-    uint8_t gas_measure;        /**< Enable gas measurement */
-    uint16_t heater_dur;        /**< Heater duration in ms */
-    uint16_t heater_temp;       /**< Heater temperature in °C */
-    uint8_t power_mode;         /**< Power mode (sleep or forced) */
-    uint8_t settings;           /**< Settings used */
-    bme680_intf_t intf;         /**< Hardware interface parameters */
+    uint8_t ambient_temp;
+    uint8_t temp_os;
+    uint8_t press_os;
+    uint8_t hum_os;
+    bool meas_gas;
+    uint16_t gas_heating_time;
+    uint16_t gas_heating_temp;
+    uint8_t filter;
+    bme680_intf_t intf;
```

Comment:
```suggestion
    bme680_intf_t intf;     /**< please document me */
    uint16_t gas_heating_time; /**< and me */
    uint16_t gas_heating_temp; /**< and me */
    uint8_t ambient_temp; /**< and me */
    uint8_t temp_os; /**< and me */
    uint8_t press_os; /**< and me */
    uint8_t hum_os; /**< and me */
    uint8_t filter; /**< and me */
    bool meas_gas; /**< and me */
```

Please add documentation to the members. (And also please align the documentation.)

In addition, it is useful to spent some thought into the memory layout of a structure. E.g. consider this code:

```C
#include <stdint.h>
#include <stdio.h>

struct a {
    uint8_t m1;
    uint32_t m2;
    uint8_t m3;
    uint32_t m4;
    uint16_t m5;
};

struct b {
    uint32_t m2;
    uint32_t m4;
    uint16_t m5;
    uint8_t m1;
    uint8_t m3;
};

int main(void)
{
    printf("sizeof(struct a) = %u, sizeof(struct b) = %u\n", (unsigned)sizeof(struct a), (unsigned)sizeof(struct b));
    return 0;
}
```

On most 32-bit and 64-bit machines this will print `sizeof(struct a) = 20, sizeof(struct b) = 12`. So `struct a` - despite having the exact same members as `struct b` - is 8 bytes larger. This is because the compiler will add padding before the struct members, if the alignment of the member is not fulfilled. A simple algorithm to yield the most efficient memory use is to order members by the size of their type. But this is not the only way to get the lowest number of padding bytes needed. Feel free to order struct members as you like, but it would be good if you could avoid padding whenever possible.

For more details, this is a good read: https://hownot2code.com/2016/11/10/the-lost-art-of-c-structure-packing/